Export magic constants during server initialization (#1179)
elkowar/eww: docs/src/magic-vars.md

The delay between all the updating variables except `EWW_TIME` is 2s, for `EWW_TIME` it is 1s.

There are also `magic constants`, marked with `(Magic constant)` after the variable's name. As the name implies, they do not change.

> You can also get them from within your scripts since they are automatically exported as environmental variables!

Reviewer: ```suggestion
You can also access them in your scripts since they are automatically exported as environmental variables!
```
